### DISCUSSION TITLE
Is development stopped?

### DISCUSSION THREAD

**Discussion by xmha97** [florisboard/florisboard#2310] [Q&A] [answered]
Is development stopped?

**Comment by M00NJ** [comment, marked as answer]
No, the app is still being developed. The dev got stuck trying to implement NLP (word suggestion) and lost motivation. On top of that he's also very busy in his personal life. However he has decided to delegate some work and delay the NLP to a later release, to get the project back on track :) See #2284

**Comment by dd-dreams** [comment]
Please see this https://github.com/florisboard/florisboard/discussions/2314

**Comment by inson1** [comment]
@xmha97 btw if you got yor answer, why dont you close this question?
